app/src/main/java/org/oppia/android/app/testing/SplashTestActivity.kt

```diff
@@ -11,7 +11,7 @@ import javax.inject.Inject
  * A test activity to verify the injection of [PlatformParameterValue] in the [SplashActivity].
  * This test activity is used in integration tests for platform parameters.
  */
-class SplashTestActivity : InjectableAppCompatActivity() {
+class SplashTestActivity : InjectableAutoLocalizedAppCompatActivity() {
```

Comment:
Is this correct? I'd expect to use system language when testing the splash screen.

Reply:
Updated.